File: PRAKTIKUM MODUL 3/PRAK301_2410817310009_Muhamad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main (){
    int a, b; 

    printf ("Input: ");
    scanf ("%d %d", &a, &b);

    if (a < b) {
        printf("Output: %d %d", a, b);
    } else if (a > b) {
        printf("Output: %d %d", b, a);
    }
    return 0;
    
}

File: PRAKTIKUM MODUL 3/PRAK301_2410817310009_Muhamad_Ghazi_Rakhmadi.py
angka1, angka2 = map(int, input("Input: ").split())

if angka1 < angka2:
    terkecil = angka1
    terbesar = angka2
elif angka1 > angka2:
    terkecil = angka2
    terbesar = angka1
else:
    terkecil = terbesar = angka1 

print(f"Output: {terkecil} {terbesar}")
